requirements/dev.txt

```diff
@@ -37,6 +37,7 @@ requests-mock==1.9.3  # https://github.com/jamielennox/requests-mock
 respx==0.19.2  # https://lundberg.github.io/respx/
 pytest==7.1.2  # https://github.com/pytest-dev/pytest
 pytest-django== 4.5.2 # https://github.com/pytest-dev/pytest-django/
+freezegun==1.2.2
```

Comment:
En rebasant, ce conflit devrait s’en aller. Freezegun a été ajouté dans `requirements/dev.in`:
https://github.com/betagouv/itou/blob/master/requirements/dev.in#L37